### ISSUE TITLE
Reprocess Batch immediately if small

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#1461]

**Comment by edickens** [comment]
I have a lot of "Reprocess Batch" to do in order to get rid of the 0 years.  But it seems that they are always set for overnight processing.  This means that I have to keep a record of each one that I have worked on and then return the next day to check that there are no other errors.  I also have to unlock many of the batches and relock them again afterwards.
It would be much easier if the reprocess could be done immediately, if only for batches of less than say 1,000 entries.  Then I will not need to keep a record and return the next day.  I can wait for the reprocessing to complete in the same way as "relocate batch".
I understand why it was done this way originally, but now things have settled down, perhaps the restriction to overnight processing could be eased.
Eric Dickens

**Comment by edickens** [comment]
So what does this message mean?
![capture](https://user-images.githubusercontent.com/8876165/40469155-7d739c66-5f27-11e8-8aee-baaaee227092.JPG)
And at what time is it run, but one did work yesterday?

**Comment by Sherlock21** [comment]
I have not seen that system message for years! ALL my batches return:-
/Users/eric/Desktop/BatchProcessign msg.jpg

**Comment by edickens** [comment]
I am using "Reprocess Batch".  All my batches, no matter how small, get set this.  And they do process, but not sure at what time.  Looks like 8am BST.

**Comment by edickens** [comment]
And there seems to be another problem with Reprocess Batch.  The zero years disappear for the Batch, but the year range displayed for the Register still shows 0.
See "Details of the Parish Register register for St Peter in Babraham of Cambridgeshire"
The 0 also still shows under the "Transcriptions" action.

**Comment by Sherlock21** [comment]
Yes, I have noticed that. The only way to get rid of it is to delete the batch, then load the corrected file as if new. Assuming your profile has the 'DELETE Everything now' action.
If not, there are 2 ways:  you can use REMOVE BATCH, wait overnight, then load you new Batch as NEW
or: REMOVE the offending Batch.  And name your replacement as a different file name, and upload it as NEW straight away. if you don't pick a new batch name then this won't work!

**Comment by edickens** [comment]
I was trying to avoid the download and upload.  Just a straight reprocess.  CAM has a lot of them to do.  I'm looking for a one button press.
I can see now why CCs are not doing it.

**Comment by Captainkirkdawson** [comment]
There was a legacy message from the physical files reprocessing submission.
There is no overnight processing.
Message changed and deployed to master
